test/expected.txt

```diff
@@ -74,7 +74,7 @@
 || Tags names should be alphanumeric
 || ===================================================
 || Enter command: || [Command entered:  add []\[;] p/12345 e/valid@e.mail a/valid, address]
-|| Person names should be spaces or alphabetic characters
+|| Person names should be spaces, hyphens or alphabetic characters
```

Comment:
Good that you've updated the I/O tests, but perhaps you can write more test commands to test that names with hyphens are accepted by the AddressBook. You can also edit AddressBookTest to change some of the names to include hyphens.